jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -22,6 +23,11 @@
             :value="item.label"
           ></v-radio>
         </v-radio-group>
+        <div v-if="!wcs_available">
+          <v-alert type='warning' style="margin-left: -12px; margin-right: -12px">

```

Comment:
I think the styling here or in the overlay container needs to be tweaked.  There should be some padding between this and the edge of the sidebar.  Or maybe just moving this above (or below) the whole spinner logic would be simpler.

Reply:
Would this be better?
![image](https://github.com/bmorris3/jdaviz/assets/7179333/b788dd0c-1097-4547-952d-a1c038dc88ee)

Reply:
yep - I actually think this makes sense on top since the user can't do _anything_ with the plugin (as it stands) in this case.